extension/dynamicconfig/service/localconfig.go

```diff
@@ -31,19 +31,20 @@ import (
 // what schedules to change. The file is read live, so changes to it will
 // reflect immediately in the configs.
 type LocalConfigBackend struct {
-	viper        *viper.Viper
+	viper *viper.Viper
```

Comment:
format

Reply:
Good point, I've been playing around with it, and it seems like contiguous lines in a struct have the same tabulation applied, but separate blocks may have different spacing. I've separated the struct into these blocks to make more clear what variables are guarded by the mutex, but it seems like `gofmt` formats per block, rather than the struct overall.